pkg/agent/agent.go

```diff
@@ -293,14 +292,6 @@ func (i *Initializer) initOpenFlowPipeline() error {
 
 	// When IPSec encyption is enabled, no flow is needed for the default tunnel interface.
```

Comment:
Not your code, but seems this comment is outdated. Maybe remove it.

Reply:
Do you mean the IPSec behavior is changed and this comment is outdated? We can remove this if it is not used.

Reply:
I think Jianjun means you can just remove the comment altogether. It doesn't match the code below. The code below just installs the default tunnel flow in case Antrea uses tunnels, which is self explanatory. It doesn't test for IPsec use.

Reply:
OK. Removed this line.